src/event/property.ts

```diff
@@ -1,4 +1,5 @@
 export interface Property {
   Brightness?: string;
   Status?: string;
+  BasicState?: string;
```

Comment:
What is the difference between the `Status` and `BasicState` property?

Reply:
This property is taken over from the Niko Hobby API documentation (see **Device model - NHC Free Start Stop Actions** on page 45)